doc/source/whatsnew/v0.18.2.txt

```diff
@@ -75,6 +75,7 @@ Other enhancements
     pd.Timestamp(year=2012, month=1, day=1, hour=8, minute=30)
 
 - The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``decimal`` option (:issue:`12933`)
+- The ``pd.read_csv()`` with ``engine='python'`` has gained support for the ``na_filter`` option (:issue:`13321`)
```

Comment:
I think we try to have separate issue on separate lines that were closed by different PR's, but yes occasionally do bunch them up if they are repetitive.